pkg/release/changelog/changelog.go

```diff
@@ -34,37 +34,37 @@ type parseParams struct {
 var knownComponentParseParams = map[string]parseParams{
 	"app-operator": {
 		tag:       "https://github.com/giantswarm/app-operator/releases/tag/v{{.Version}}",
-		changelog: "https://raw.githubusercontent.com/giantswarm/app-operator/master/CHANGELOG.md",
+		changelog: "https://raw.githubusercontent.com/giantswarm/app-operator/v{{.Version}}/CHANGELOG.md",
```

Comment:
Hold on - does this actually work? Does the tagged version already have the correct patterns?

Reply:
It worked for aws-operator: https://github.com/giantswarm/releases/pull/507